fastlane/metadata/steam/german/description.txt

```diff
@@ -3,11 +3,13 @@
 [h2]Spielmechanik[/h2]
 
 [list]
-[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen
-[*] Stelle Materialien her, um deine Fabrik zu vergrößern und sie vor Gegnerwellen zu schützen
-[*] Spiele mit deinen Freunden in plattformübergreifenden Partien oder fordere sie zum Kampf heraus
-[*] Es gibt immer etwas zu tun: Wenn du deine Kühlsysteme nicht im Blick behältst, könnte dein Reaktor explodieren!
-[*] Produziere Einheiten, die bei deiner Basis helfen oder den Gegner angreifen
+[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen und eine große Vielfalt an Ressourcen herzustellen
+[*] Benutze Produktionsblöcke, um eine große Vielfalt an Ressourcen herzustellen
+[*] Beschütze deine Basis vor Gegnerwellen
+[*] Spiele mit deinen Freunden in plattformübergreifenden Partien, oder fordere sie zu einem Team-basiertem PvP-Spiel hinaus
+[*] Verteile Flüssigkeiten und bekämpfe die ständige Bedrohung von Feuerausbrüchen oder gegnerische Luftangriffe
```

Comment:
```suggestion
[*] Transportiere Flüssigkeiten zu Gebäuten und bekämpfe die ständige Bedrohung von Feuerausbrüchen oder gegnerische Luftangriffe
```
Verteile Flüssigkeiten klingt cursed, habe keine deutsche Beispiele dazu gefunden wie es im Rohrbau so genutzt wird, auch in Jobausschreibungen wird es in der deutschen Übersetzung komplett vermieden durch umschreiben